tests/test_filter_datasets.py

```diff
@@ -294,3 +341,109 @@ def test_stratified_crossval_splits_multiple_attributes(many_stimuli, crossval_f
     assert sum(len(f.x) for f in train_fixations) == (crossval_folds - val_folds - test_folds) * len(fixations.x)
 
     assert len(train_stimuli) == crossval_folds
+
+
+@pytest.mark.parametrize('attribute_name , attribute_value, invert_match', [('dva', 1, False)])
+def test_filter_stimuli_by_attribute_dva(file_stimuli_with_attributes, fixation_trains, attribute_name, attribute_value, invert_match):
+    fixations = fixation_trains[:]
+    filtered_stimuli, filtered_fixations = filter_stimuli_by_attribute(file_stimuli_with_attributes, fixations, attribute_name, attribute_value, invert_match)
+    if attribute_name == 'dva' and attribute_value == 1 and invert_match is False:
+        inds = [1]
+        expected_stimuli, expected_fixations = create_subset(file_stimuli_with_attributes, fixations, inds)
+        compare_fixations(filtered_fixations, expected_fixations)
+        assert list(filtered_stimuli.stimulus_ids) == file_stimuli_with_attributes.stimulus_ids[inds]
+        assert_stimuli_equal(filtered_stimuli, expected_stimuli)
+
+
+@pytest.mark.parametrize('attribute_name , attribute_value, invert_match', [('some_strings', 'n', True)])
+def test_filter_stimuli_by_attribute_some_strings_invert_match(file_stimuli_with_attributes, fixation_trains, attribute_name, attribute_value, invert_match):
+    fixations = fixation_trains[:]
+    filtered_stimuli, filtered_fixations = filter_stimuli_by_attribute(file_stimuli_with_attributes, fixations, attribute_name, attribute_value, invert_match)
+    if attribute_name == 'some_strings' and attribute_value == 'n' and invert_match is True:
+        inds = list(range(0, 13))+list(range(14, 18))
```

Comment:
sorry, I got really used to PEP8 ;). I like that it makes the code a bit less dense.

```suggestion
        inds = list(range(0, 13)) + list(range(14, 18))
```

Reply:
fixed